compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionOfTensorOps.cpp

```diff
@@ -362,6 +362,11 @@ struct FusionOfTensorOpsPass
               return false;
             }
 
+            // Do not fuse by expand if consumer is dequant.
+            if (isGroupedDequantizationOp(consumer)) {
```

Comment:
So wondering if we need this. These patterns are meant to propagate the reshape up. So typically you have the expand shape as the consumer and generic op as producer. For the dequantization op, you want to propagate this to help with fusion later. Do you know why we need this, i.e. an example?

Reply:
I am not sure. It comes from @raikonenfnu's changes. Perhaps he could elaborate on why it is needed here?

Reply:
we have an example in the fusion_of_tensor_ops.mlir test. I will review a bit first before our meet to brush off my vacation brain :D